app.py

```diff
@@ -142,7 +142,7 @@ def index_separate_iframes():
         # serve up the auth iframe to start
         request_token = ''.join(random.choice(string.lowercase + string.digits) for i in range(15))
         session['ToopherRequestToken'] = request_token
-        auth_iframe_url = api.auth_uri(username, reset_email, action, automation_allowed, challenge_required, request_token, requester_metadata, ttl);
+        auth_iframe_url = api.get_auth_url(username, reset_email, request_token, action, requester_metadata, automation_allowed=automation_allowed, challenge_required=challenge_required, ttl=ttl);
```

Comment:
How do you feel about renaming the method `get_auth_iframe_url`? The client is assigning `auth_iframe_url` to the output of that method, so it seems that the method should be getting the auth iframe url. Thoughts as the one who has to understand this client code?